csharp/lib/glide.csproj

```diff
@@ -9,19 +9,35 @@
   </PropertyGroup>
 
   <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
-    <Exec Command="cargo build --release" />
+    <Exec Condition="'$(Configuration)' == 'Debug'" Command="cargo build" />
+    <Exec Condition="'$(Configuration)' == 'Release'" Command="cargo build --release" />
   </Target>
 
   <ItemGroup>
-    <Content Include="$(MSBuildThisFileDirectory)target/release/libglide_rs.dll" Condition="$([MSBuild]::IsOSPlatform('Windows'))">
```

Comment:
I think we should use path $(MSBuildThisFileDirectory)target/$(Configuration.ToLower())/glide_rs.dll to have less branches